### ISSUE TITLE
Example

### ISSUE THREAD

**Issue by ovninosa** [Pooja199/SpeechClassification#1]
Hi,

Can you make an example to work with this usings wavs?

Thanks
Jave

**Comment by Pooja199** [comment]
Can you be more specific, so i can explain it better.

Thanks
Pooja

**Comment by ovninosa** [comment]
Yes, I want to test your code with a voices dataset audio files (wav) to analyze the genuinity.
The evaluation result is a score or boolean?

Can you make an example with that? or at least a guide?
I see a few paths on your code but are related to your pc.

Thanks
Jave

**Comment by ovninosa** [comment]
@Pooja199 can you help me with this?

Thanks!

**Comment by Pooja199** [comment]
The result is a score.
You first perform cqcc though which you get the scores for your training
data set.
Use this data (path) for your check.

Regards,
Pooja

**Comment by ovninosa** [comment]
@Pooja199 I saw this paths over your code.

With the files of this repo is enough to test it? Or I need another file from your personal pc?

Thanks

			% set paths to the wave files and protocols
			pathToDatabase = fullfile('C:\Users\Pooja\Desktop\baseline_CM','ASVspoof2017_train_dev');
			trainProtocolFile = fullfile('C:\Users\Pooja\Desktop\baseline_CM\ASVspoof2017_train_dev', 'protocol', 'ASVspoof2017_train.trn');
			devProtocolFile = fullfile('C:\Users\Pooja\Desktop\baseline_CM\ASVspoof2017_train_dev', 'protocol', 'ASVspoof2017_dev.trl');

			pathToDatabase = fullfile('C:\Users\Shrirag\Desktop','ASV Trimmed Data');

			evalProtocolFile = fullfile('C:\Users\Shrirag\Desktop','ASV Trimmed Data','ASVspoof2017_eval.trl');

**Comment by Pooja199** [comment]
Hey. This is enough
